scripts/preview/download-and-merge-harvester-kubeconfig.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
Note to self: We want to exit early here if the workspace isn't from a Gitpod employee (e.g. we have quite a few assumptions about the environment, so we should exit early and cleanly so that other people opening workspaces don't just get a ton of errors)